### ISSUE TITLE
Combustion Chamber Stops Working until Server Restart

### ISSUE THREAD

**Issue by ghost** [Bartz24/SkyResources#130]
We're running the Sky Resources 2 Mod on a private server locally hosted.  Over the past couple of days, we've been having an issue where the combustion chamber will not actually generate new materials until the server is restarted.  We will see and hear the visual/audio cues for it, but no materials are destroyed/created, and no fuel is used. This happens regardless of the resource type that we are trying to generate.  The only fix thus far is to restart the server, and this will only work for 1-2 runs through the system until it will need to be restarted again.

There are no errors in the server as far as I can see when this happens. Any help on this would be great.

**Comment by Kherune** [comment]
Make a small correction there is no actual sound either, simply the visual animation of the explosion. otherwise all other details are correct.

The issue seems to have occurred soon after we got to the stage where we could automate the process. Adding a redstone clock the Quick Dropper and The Combustion Collector is when we ran into the issue.

I'm currently testing to see if we can find results and figure out a solution, I will post with updates so future googlers have a solution should I find one.

Update: Now its just straight broken after the restart went through which had temporarily fixed the issue before it does not seem to matter now. So back to square one.
